### PR TITLE
fix package versions

### PR DESCRIPTION
**Pull request by chainyo**
I fixed package versions to make it more stable while people use the package.

I tried to use the lastest version as base version for every package.

### CODE REVIEW

**Comment by AlekseyKorshuk** [comment]
Looks good. But PyPi integration may crash because "requirements.txt was not found"

**Comment by chainyo** [comment]
It seems that `matplotlib` is also required by the package because it is used in `optimum_transformers/utils/benchmark.py`.

**Comment by chainyo** [comment]
> Looks good. But PyPi integration may crash because "requirements.txt was not found"

Do you get this error while doing `pip install .` or something else ? @AlekseyKorshuk

**Comment by AlekseyKorshuk** [comment]
In GitHub Actions

File: setup.py [hunk 1]
```diff
@@ -29,7 +18,7 @@
               "infinity"],
     license="Apache",
     url="https://github.com/AlekseyKorshuk/optimum-transformers",
-    install_requires=requirements,
+    install_requires=open("requirements.txt", "r", encoding="utf-8").read().splitlines(),
```
Comment (chainyo) [inline setup.py:21]: Ok I have added the `open` function directly to the setup. Is it resolving the file problem ?

If not I saw some changes to bring to the ci pipeline to make it work with a requirements.txt file, or we could directly declare dependencies in the setup.py file and remove the requirements.txt file.

What do you think ?

Comment (AlekseyKorshuk) [reply setup.py:21]: Lets give it a try

**Review by AlekseyKorshuk — APPROVED** [review]

**Pull request merged**